tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "./tsconfig.base.json",
```

Comment:
why not put this in the base config?

Reply:
> why not put this in the base config?

Each tsconfig has different settings and looks at different files. For the caching to be effective, each tsconfig should have it's own `tsBuildInfoFile` . Technically we could add 'incremental' to base and then add the `tsBuildInfoFile` in each tsconfig but it doesn't seem nice because those two keys have to be together - it's an invalid config to have `incremental` without `tsBuildInfoFile`.

Given the above it seems cleaner to specify it in each tsconfig separately, so I just started with the main one as a minimal change